ext/ffi/lib.rs

```diff
@@ -1031,40 +1027,37 @@ fn ffi_parse_pointer_arg(
       })?
       .get_backing_store();
     if byte_offset > 0 {
```

Comment:
From what I can tell, `byte_offset` is `usize` - unsigned may below zero?
```suggestion
backing_store[byte_offset..].as_ptr().cast::<c_void>()
```

Reply:
Should you commit this suggestion, this structure crops up a few other times.

Reply:
As mentioned elsewhere, this is checking for non-zero and is related to slice access beyond limits for zero-length slices.

Reply:
Err, can you double-check this? I was able to run `&[u8::MIN; 0][0..]` without issue locally in debug mode without optimisations applied.

Reply:
Hum... I am absolutely sure that this was causing panics earlier but now it's somehow fine?